### DISCUSSION TITLE
Future plans: local or online focus?

### DISCUSSION THREAD

**Discussion by goddaneel** [Acclorite/book-story#40] [Q&A]
I see networking permissions in the permissions request and was wondering if this software is planned to be a locally focused book reader or if it will add more online networking capabilities.

**Comment by Acclorite** [comment]
Please, if your issue does not contain Feature Request, Bug report or anything similar, start a [Discussion](https://github.com/Acclorite/book-story/discussions) instead.

No the app will not focus on online. The only reason for network permission is "Check for updates" feature, which retrieves latest release information from **api.github.com**. I don't want my app to depend on any network service.

**Reply by goddaneel** [reply]
Understood, and I apologize if the inappropriate issue has caused you distress.

The reason I had this query is that some of the previous reader programs I've used have had a terrible local reading experience due to overreaching for online features. I'm glad to hear that you plan to focus on local.

**Reply by Acclorite** [reply]
Haha, I am not so insane yet to be distressed about wrong category, it's fine :)

**Comment by shuvashish76** [comment]
https://github.com/Acclorite/book-story#-features

On readme you mentioned
> Check for app updates on app startup

Is that still holds true? Or only when the user click on check for updates from About screen?

> Double click translation(sends to the external translator)

Only external i.e. installed applications right? No inbuilt translator feature correct?

**Reply by Acclorite** [reply]
> Is that still holds true? Or only when the user click on check for updates from About screen?

Yes it still works perfectly fine if you enable it from General settings (and it also works in About screen).

> Only external i.e. installed applications right? No inbuilt translator feature correct?

Yes, as fast as I discovered Built-in Translator was closed source, it was deleted. Now on double click it sends you via intent to another app. If translator has floating window feature, then it will open just on top of this app, which is pretty convenient.

**Reply by shuvashish76** [reply]
> Yes it still works perfectly fine if you enable it from General settings

So it's opt-in i.e. disabled by default, nice 👍🏿

> Now on double click it sends you via intent to another app.

I think perform actions via "context menu" is better than this. Requested at #46
